ansible/roles/rancher/vars/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
 cert_manager_version: v0.9.1
 rancher_password: "{{ lookup('env','RANCHER_PASSWORD') }}"
-rancher_version: 2.2.8
\ No newline at end of file
+rancher_version: 2.3.0
```

Comment:
Can we do the latest [2.3.2](https://github.com/rancher/rancher/releases/tag/v2.3.2)?